core/ui/EditTemplate.tid

```diff
@@ -1,28 +1,28 @@
 title: $:/core/ui/EditTemplate
 
-\define actions()
-<$action-sendmessage $message="tm-add-tag" $param={{$:/temp/NewTagName}}/>
-<$action-deletetiddler $tiddler="$:/temp/NewTagName"/>
-<$action-sendmessage $message="tm-add-field" $name={{$:/temp/newfieldname}} $value={{$:/temp/newfieldvalue}}/>
-<$action-deletetiddler $tiddler="$:/temp/newfieldname"/>
-<$action-deletetiddler $tiddler="$:/temp/newfieldvalue"/>
+\define save-tiddler-actions()
+<$action-sendmessage $message="tm-add-tag" $param={{{ [<newTagNameTiddler>get[text]] }}}/>
```

Comment:
Apologies if we've covered this before, but why has `{{$:/temp/NewTagName}}` been changed to `{{{ [<newTagNameTiddler>get[text]] }}}`?

Reply:
That's because the input tiddlers are qualified tiddlers